src/services/dynamo-database-service.ts

```diff
@@ -103,7 +103,7 @@ export class DynamoDatabaseService {
       return response;
     } catch (error: any) {
       if (!error.name || error.name !== 'ConditionalCheckFailedException') {
-        logger.error(`${LOGS_PREFIX_SENSITIVE_INFO} Error updating item with pk ${userId}.`);
+        logger.error(`${LOGS_PREFIX_SENSITIVE_INFO} Error updating Dynamo DB ${error}.`);
```

Comment:
could we print the message and include error as a object

Reply:
It doesn't like it because the error is of type unknown/any

Reply:
Hi @jessica-li-01 - see latest commit